cadquery/occ_impl/shapes.py

```diff
@@ -2371,45 +2373,38 @@ def shell(
         }
 
         occ_faces_list = TopTools_ListOfShape()
+        shell_builder = BRepOffsetAPI_MakeThickSolid()
 
-        if faceList:
```

Comment:
Why did you remove this check to make sure faceList is not None? I see that same check used below.

Reply:
I was going by the type hinting on the signature (`faceList: Iterable[Face],`), so I didn't think `None` was an option.
But you're right, the way that code was originally written it would work with `None` so I should probably change the type hinting to `Optional[Iterable[Face]]` and handle it.

Reply:
e41b0300138350e50870292fedf63b606587e68a should take care of this.